server/src/domain/person/person.service.ts

```diff
@@ -84,6 +84,16 @@ export class PersonService {
     return this.findOrFail(id).then(mapPerson);
   }
 
+  async getPersonAssetsCount(authUser: AuthUserDto, id: string): Promise<number> {
+    await this.access.requirePermission(authUser, Permission.PERSON_READ, id);
+    const person = await this.repository.getById(id);
+    if (!person) {
+      throw new NotFoundException();
```

Comment:
We're currently using bad requests for these situations right now. Also, just use `this.findOrFail()`. Actually, you don't even need this check, since the access check will fail if the person doesn't exist and you don't use the person for anything anyways.